conda/recipes/libcudf/meta.yaml

```
@@ -195,7 +195,7 @@ outputs:
       license: Apache-2.0
       license_family: APACHE
       license_file: LICENSE
-      summary: libcudf_example library
+      summary: libcudf executable examples
```

Match the naming of `libcudf-tests`, which says "libcudf test & benchmark executables"
```suggestion
      summary: libcudf example executables
```

---

All done!